addon/components/horizontal-list-swiper/gesture2/styles.less

```diff
@@ -6,6 +6,9 @@
     list-style: none;
     margin: 0;
     padding: 0;
+    transition-property: all;
+    transition-timing-function: ease;
+    transition-delay: 0s;
```

Comment:
Similar comment for this one:
`transition: all ease 0s`

Reply:
same as above